devicon.json

```diff
@@ -3719,6 +3719,28 @@
         "color": "#000000",
         "aliases": []
     },
+    {
+        "name": "insomnia",
+        "altnames": [
+            "Insomnia Rest",
+            "Insomnia Rest Client"
+        ],
+        "tags": [
+            "open-source",
+            "Rest API",
+            "Soap API",
+            "JSON-XML"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark"
+            ],
+            "font": []
```

Comment:
`plain` and `plain-wordmark` should be added to `version.svg[]` and `version.font[]` after `original` and `original-wordmark` are fixed to following the SVG standards. 📝